File: Main/brain/DiveMaster.rb
class DiveMaster
	def initialize
		@NEUTRAL = 0
		@FULL_AHEAD = 0.99
		@FULL_REVERSE = -0.99
	end
	def setForwardThrust(velocity)
		print "setForwardThrust #{velocity}"
	end
	def setDiveThrust(velocity)
		print "setDiveThrust #{velocity}"
	end
	def setStrafeThrust(velocity)
		print "setStrafeThrust #{velocity}"
	end
	def setYawThrust(velocity)
		print "setYawThrust #{velocity}"
	end
end

File: Main/brain/DiveMaster.js
function DiveMaster() {
	this.NEUTRAL = 0;
	this.FULL_AHEAD = 0.99;
	this.FULL_REVERSE = -0.99;

	this.setForwardThrust = function(velocity) {
		console.log("setForwardThrust " + velocity);
	};
	this.setDiveThrust = function(velocity) {
		console.log("setDiveThrust " + velocity);
	};
	this.setStrafeThrust = function(velocity) {
		console.log("setStrafeThrust " + velocity);
	};
	this.setYawThrust = function(velocity) {
		console.log("setYawThrust " + velocity);
	};
}

module.exports = DiveMaster;
